packages/frontend/src/components/Search/FilterSearchData.tsx

```diff
@@ -0,0 +1,121 @@
+import { flatMapByKey, generateYearsBetween } from '../../utils/helpers';
+import categoryData from '../../assets/data/aineistoluokka.json';
+
+export enum ItemTypeEnum {
+  CHECKBOX = 'checkbox',
+}
+
+export interface IItem {
+  name: string;
+  type?: ItemTypeEnum;
+  items?: IItem[];
+}
+
+export interface IFilterSearchData {
```

Comment:
This and `IItem` have the same contents. Do they need to be separate? Can one maybe extend other?